src/iconsManifest/supportedExtensions.ts

```diff
@@ -2622,12 +2628,6 @@ export const extensions: IFileCollection = {
       format: FileFormat.svg,
     },
     { icon: 'lync', extensions: ['crec', 'ocrec'], format: FileFormat.svg },
-    {
-      icon: 'makefile',
-      extensions: ['makefile'],
```

Comment:
Why did you remove the `makefile` extension?

Reply:
As in the PR commment:
> Rename `makefile` logo -> `gnu` & add `.ld` extension to it

Reply:
I still don't understand why the `makefile` extension has been removed. I think we should keep supporting that and don't introduce a breaking change.

Reply:
True! It's because the icon for `makefile` is actually the `gnu` logo, so there'd be a double-up.

Reply:
Yes, but I'm talking about the `makefile` extension. Line 2638.  My guess is that it should be added to ln 1903.